pymchelper/utils/mcscripter.py

```diff
@@ -0,0 +1,303 @@
+"""
+Tool for creating MC input files using user-specified tables and ranges.
+
+2019 - Niels Bassler
+"""
+
+import os
+import sys
+import errno
+import logging
+
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+class Config():
+    """
+    Reading the config file.
+    """
+    def __init__(self, fn):
+        with open(fn) as _f:
+            self.lines = _f.readlines()
+            self.base_dir = os.path.dirname(fn)
+
+            # script must run relative to location of config file.
+            if self.base_dir:
+                os.chdir(self.base_dir)
```

Comment:
Why a change dir here ? Is it really needed ?

Reply:
i am afraid so, `os.symlink()`must make relative symlinks, and when they are created outside of the dir, `os.symlink()` will complain that the target do not exist.
Giving paths to files relative to non-base positions will make `os.symlink()` work, but then the links are invalid instead.

I added a few comments in the next commit.